syne_tune/optimizer/schedulers/searchers/bayesopt/datatypes/common.py

```diff
@@ -30,6 +30,9 @@ def dictionarize_objective(x):
 Configuration = Dict[str, Hyperparameter]
 
 
+ConfigurationFilter = Callable[[Configuration], bool]
```

Comment:
Would be useful to add 1-2 sentence comment on the context and how its used for warm-starting.

Reply:
OK